test.sh

```diff
@@ -97,12 +93,39 @@ function setup_envs {
   export TEST_ASSET_ETCD=$kb_root_dir/bin/etcd
 }
 
+function is_installed {
+  if [ command -v $1 &>/dev/null ]; then
+    return 0
+  fi
+  return 1
+}
+
+function fetch_go_tools {
+  header_text "Checking for golangci-lint"
+  if ! is_installed golangci-lint; then
+    GO111MODULE=on go get -d github.com/golangci/golangci-lint/cmd/golangci-lint@v1.15.0 
+  fi
+
+
+  header_text "Checking for dep"
+  if ! is_installed dep; then
+    header_text "Installing dep"
+    GO111MODULE=off go get -d -u github.com/golang/dep
+    cd $(go env GOPATH)/src/github.com/golang/dep
```

Comment:
I don't think we should pull latest here -- I think we should just download the a specific dep version as per `GOOS` and `GOARCH` -- it makes things more reproducible.